features/step_definitions/secret_restore.rb

```diff
@@ -0,0 +1,13 @@
+Given /^admin ensures the changed secret "([^"]*)" is restored in "([^"]*)" project after scenario$/ do | secret , project |
```

Comment:
No need to add a new step. We already have a step `/^the #{QUOTED} (\w+) is recreated( by admin)? in the#{OPT_QUOTED} project after scenario$/`. Somehting like:
```
Given the "vsphere-creds" secret is recreated by admin in the "kube-system" project after scenario
```

Reply:
Thank you , reused and validated , looks good 
https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/183113/console